Source/ACE.Server/Factories/LootGenerationFactory.cs

```diff
@@ -902,9 +901,8 @@ private static double GetMaxDamageMod(int tier, int maxDamageMod)
                 default:
                     break;
             }
-            double damageMod2 = 1.0 + damageMod;
 
-            return damageMod2;
+            return damageMod + 1;   
```

Comment:
nbd, but since damageMod is an additive float multiplier, the 1.0 + value format seemed clearer imo